tests/test_project/tests/test_admin.py

```diff
@@ -53,3 +53,14 @@ def test_alwayshaschangedmixin(self):
         self.assertEqual(Operator.objects.count(), 1)
         project = Project.objects.first()
         self.assertEqual(project.name, params['name'])
+
+    def test_custom_admin_site(self):
+        url = reverse('admin:logout')
+        response = self.client.get(url)
+        content = str(response.content)
+        # Login again for next tests
+        user = User.objects.get(username='administrator')
```

Comment:
In my sample I used `logout` template just because I don't have to do a lot of work.
But for the final work, can you please use something like `password_change_template` to avoid this relogin? 

Thanks.